alg/teca_dataset_diff.h

```diff
@@ -101,7 +114,7 @@ class teca_dataset_diff : public teca_algorithm
         const teca_metadata &request) override;
 
 private:
-    // Tolerance for equality of field values.
+    /** Tolerance for equality of field values. */
```

Comment:
this is documented via the algorithm property, we should probably remove this one